src/gt4sd/algorithms/generation/diffusion/core.py

```diff
@@ -127,12 +127,12 @@ class DiffusersConfiguration(AlgorithmConfiguration[str, None]):
         ),
     )
 
-    prompt: str = field(  # type: ignore
+    prompt: Union[str, Dict[str, Any]] = field(  # type: ignore
         default=None,
-        metadata=dict(description="Prompt for text generation."),
+        metadata=dict(description="Prompt for conditional generation."),
     )
 
-    def get_target_description(self) -> Optional[Dict[str, str]]:
+    def get_target_description(self) -> Optional[Any]:
```

Comment:
we should remove this method here to ensure we use the generic one coming from `AlgorithmConfiguration`